Makefile

```diff
@@ -20,7 +20,8 @@ CONCORD_BFT_CMAKE_FLAGS:= \
 			-DUSE_S3_OBJECT_STORE=TRUE \
 			-DUSE_OPENTRACING=ON \
 			-DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
-			-DOMIT_TEST_OUTPUT=OFF
+			-DOMIT_TEST_OUTPUT=OFF \
+			-DKEEP_APOLLO_LOGS=TRUE
```

Comment:
We should "KEEP_APOLLO_LOGS" only in CI. When running locally, it should be up to the developer to activate this or not.

Reply:
@teoparvanov , IMO, it is better to turn on, so that a developer would have the same logs as in the CI.
If somebody does not need it, they may disable it explicitly.